styles.css

```diff
@@ -1,67 +1,64 @@
-/* font-family: 'Quicksand', sans-serif;
-COLORS
-light-yellow - #f7e4bf;
-light-blue - #78a7c6;
-dark-blue - #134d71;
-white - #ffffff;
-black - #000000;
-*/
 
-body {
-  background-image: linear-gradient(#134d71, #78a7c6, #f7e4bf);
-  font-family: 'Quicksand', sans-serif;
-  text-align: center;
+html {
+  background: linear-gradient(#134d71, #78a7c6, #f7e4bf);
   color: #ffffff;
-  height: auto;
-  width: auto;
-  position: relative;
+  font-family: 'Quicksand', sans-serif;
+  width: 100%;
+  height: 100%;
 }
 
-form {
-  margin-left: auto;
-  margin-right: auto;
-  background: #ffffff;
-  height: 100px;
-  width: 500px;
-  border-radius: 10px 10px 10px 10px;
+.wrapper {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
+  height: 100%;
 }
 
-.choose-message-buttons {
-  columns: 2;
+.box {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  border-radius: 10px;
 }
 
-#message-box {
-  margin-left: auto;
-  margin-right: auto;
-  background: #ffffff;
-  color: #000000;
-  height: 150px;
-  width: 800px;
-  border-radius: 10px 10px 10px 10px;
-  position: relative;
-  text-align: center;
-  vertical-align: middle;
+.flex-row {
```

Comment:
Could not figure out how to get this flexbox formatting to apply to the radio buttons inside this div -- so not sure that the css for this class is doing anything.